src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -153,19 +158,41 @@ const VideoPlayer: FC<VideoPlayerProps> = (props) => {
     }
   };
   const onError = (evt: Event) => {
+    const networkError =
+      (evt as CustomEvent).detail?.data?.type === "networkError";
+    // Don't report network errors.
+    if (networkError) {
+      return;
+    }
```

Comment:
This was already the behaviour, I just moved it to the top of function. 

Not sure if anyone has any thoughts on this? I wondered if it could hide some larger connectivity issues or bandwidth issues in schools/homes. I would think Product would still like to know if there are a lots of network issues when users are viewing videos.

Reply:
lets log this an event in posthog ?

Reply:
@kimon-satan I'll ask product.

Reply:
Mentioned to @aakeshp, we'll put in a separate ticket if required.